web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v441/migrate-default.sql

```diff
@@ -1,2 +1,6 @@
 UPDATE Settings SET value='4.4.1' WHERE name='system/platform/version';
 UPDATE Settings SET value='0' WHERE name='system/platform/subVersion';
+
+UPDATE settings SET name='metadata/history/enabled' WHERE name='system/metadata/history/enabled'
```

Comment:
You need to move these changes to /migrate/v442/migrate-default.sql

Reply:
moved